Content.Server/Advertise/Components/AdvertiseComponent.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Server.Advertise.EntitySystems;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Advertise.Components;
+
+[RegisterComponent, Access(typeof(AdvertiseSystem))]
+public sealed partial class AdvertiseComponent : Component
```

Comment:
no comment: sent to gulag

Reply:
Hey now, don't blame me; I just moved the file to a different directory!

But yeah, I'll fix up the old junk in the file while I'm here.